### PR TITLE
Implement `envtool shell read` subcommand

### LINKED ISSUE #2615: Implement `envtool shell read` subcommand
### What should be done?

It should read the given file and print its content to stdout.

Then use it instead of `cat` command in the Taskfiles.

### Where?

* https://github.com/FerretDB/FerretDB/blob/main/Taskfile.yml
* https://github.com/FerretDB/FerretDB/tree/main/cmd/envtool

### Definition of Done

* envtool updated;
* Taskfile.yml updated.

### PR DESCRIPTION
**Pull request by wqhhust**
# Description

Closes #2615.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: cmd/envtool/envtool.go [hunk 1]
```diff
@@ -347,6 +348,40 @@ func rmdir(paths ...string) error {
 	return errs
 }
 
+// cat will show the content of a file.
+func cat(paths ...string) error {
+	currentDir, err := os.Getwd()
```
Comment (AlekSi) [inline cmd/envtool/envtool.go:353]: There is no need to do that – `os.Open` opens files in the current directory by default

File: cmd/envtool/envtool.go [hunk 2]
```diff
@@ -347,6 +348,40 @@ func rmdir(paths ...string) error {
...
+		panic(err)
+	}
+
+	var errs error
+
+	for _, path := range paths {
+		if !strings.HasPrefix(path, "/") {
+			path = currentDir + "/" + path
+		}
+		f, err := os.Open(path)
```
Comment (AlekSi) [inline cmd/envtool/envtool.go:364]: Let's just use `os.ReadFile` followed by `fmt.Print(string(b))`

**Review by AlekSi — COMMENTED** [review]
Let's rename `cat` to `read` everywhere

**Comment by wqhhust** [comment]
done

image

**Comment by AlekSi** [comment]
Please run `task fmt-yaml`, the re-request review (see the last point [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review)).

**Review by rumyantseva — CHANGES_REQUESTED** [review]
With the current approach `task pacakges` doesn't work anymore. I tried it on Mac, and it failed. But when I run `task packages` from the `main` branch, it works. So, it needs to be fixed.

Screenshot 2023-05-16 at 13 10 01

**Comment by wqhhust** [comment]
Added back the sh.  thx for your review

**Review by rumyantseva — APPROVED** [review]
Thanks for contributing! The proposed approach looks good to me.

I renamed the PR title, so it matches the issue title - we describe the command we implement.

**Review by w84thesun — APPROVED** [review]
LGTM!

**Review by AlekSi — APPROVED** [review]

**Pull request merged**